### DISCUSSION TITLE
Advent Calendarについての要望

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#416] [要望・アイデア・不具合]
去年のAdvent Calendarでいくつかこうなってくれたらいいな、と思ったことがあるので、要望として書かせていただきます。

* 今年以降も「完走賞」のようなものを実施するのであれば、カテゴリーを分けるか、あるいはマークのようなものをつけられるようにしてほしいです。「完走賞」狙いのカレンダーなのかそうでないのかがわからないと参加時に不安になることもあると思います。
* 同じような話ですが、カレンダー一覧上で、「完走賞」用のカレンダーを非表示にする、あるいは「完走賞」用のカレンダーだけを表示するような機能が欲しいです。例えば、参加できそうなカレンダーを探すときには「完走賞」用のカレンダーを非表示にしたほうが見つけやすいですし、「完走賞」を目指して頑張っている人の記事を見たい場合には、「完走賞」用のカレンダーだけを表示できるとよいと思います。
* カレンダー一覧では、登録済の日がある分だけ赤いバーが伸びていくような表現になっていると思います。もし参加登録済でも、記事が登録されていない場合には、色を変えることはできないでしょうか？（例：20日分登録されているけど、3日分が予定日をすぎてもまだ記事が登録されていない場合には、17日分の赤のバー、3日分の別の色のバー、空白が5日分、というように表示するなど）　「代わりに投稿」ができるかどうかがわかりやすくなると思うので。
* 通常の記事の検索画面で「Advent Calendar 2022」の参加記事、みたいな検索条件が欲しいです（見えないタグが付いているイメージか、専用の絞り込み条件の新設か、なにか）

ご一考いただけると幸いです。

**Comment by tanaka-asami415** [comment]
@yoshi389111 
詳細にありがとうございます！🙇
いただいたご意見も踏まえて、次回も改善していきます！

**Comment by tsuzuki-takaaki** [comment]
@yoshi389111
報告が遅くなってしまい、申し訳ございません🙇‍♂️

ご提案いただいていた、

> - 今年以降も「完走賞」のようなものを実施するのであれば、カテゴリーを分けるか、あるいはマークのようなものをつけられるようにしてほしいです。「完走賞」狙いのカレンダーなのかそうでないのかがわからないと参加時に不安に なることもあると思います。
> - 同じような話ですが、カレンダー一覧上で、「完走賞」用のカレンダーを非表示にする、あるいは「完走賞」用のカレンダーだけを表示するような機能が欲しいです。例えば、参加できそうなカレンダーを探すときには「完走賞」用の カレンダーを非表示にしたほうが見つけやすいですし、「完走賞」を目指して頑張っている人の記事を見たい場合には、「完走賞」用のカレンダーだけを表示できるとよいと思います。

に関してですが、Advent Calendar 2025 から「ひとりアドベントカレンダー」のカテゴリーを新設しました！
https://qiita.com/advent-calendar/2025/categories/solo_advent_calendar

他にいただいている提案に関しても、貴重なご意見として参考にさせていただき、引き続き改善を行って行きます！
よろしくお願いします🙇‍♂️
